code/frameworks/nextjs/src/swc/loader.ts

```diff
@@ -49,7 +53,7 @@ export const configureSWCLoader = async (
         hasReactRefresh: isDevelopment,
         jsConfig,
         nextConfig,
-        supportedBrowsers: require('next/dist/build/utils').getSupportedBrowsers(
+        supportedBrowsers: (await import('next/dist/build/utils.js')).getSupportedBrowsers(
```

Comment:
this can just be a regular import

Reply:
Changing this to a regular import causes this:
```
SB_BUILDER-WEBPACK5_0002 (WebpackInvocationError): Module not found: Error: Can't resolve '@angular/animations/browser' in './node_modules/@angular/platform-browser/fesm2022'
```

See: https://app.circleci.com/pipelines/github/storybookjs/storybook/101986/workflows/dff52e38-ac65-42a7-b329-e1064718911f/jobs/868016